domain-ee/ee-ep-merge-app/integration/mq_endpoint.py

```diff
@@ -4,26 +4,30 @@
 from hoppy import async_consumer, async_publisher
 from hoppy.hoppy_properties import ExchangeProperties, QueueProperties
 from pika import BasicProperties
+from src.python_src.config import ClientName
 
 
 class MqEndpointConsumerException(Exception):
     pass
 
 
 class MqEndpoint:
-    def __init__(self, name, exchange, req_queue, response_queue):
+    def __init__(self, name, exchange, req_queue, response_queue, arguments = {}):
         self.name = name
         self.index = 0
         self.auto_response_files = []
+        type = 'direct'
+        if (name == ClientName.DEAD_LETTER):
```

Comment:
Won't need this logic since this app wont need to access the DLQ.

Reply:
Same as above